### ISSUE TITLE
Accessibility : hiding non-informative SVG icons

### ISSUE THREAD

**Issue by Agesse** [stephanrauh/ngx-extended-pdf-viewer#2799]
Per WCAG 1.1.1 (https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) non text content should be hidden from users.

In many buttons, there's an `aria-label` on the button element, and a `role="img"` + `aria-label` on the SVG icon used in the button.
Only one label should be present, the button label being the one that will be read by assistive technologies.
So every SVG icon should have the following attribute `aria-hidden="true" focusable="false"` to prevent some assistive technologies from reading two labels. The `focusable`attribute is a foolproof attribute often used on SVG icons to prevent focus on the icon on IE.

https://www.accede-web.com/en/guidelines/html-css/images-and-icons/managing-alternative-text-on-svg-scalable-vector-graphics/
https://html-validate.org/rules/svg-focusable.html

**Comment by stephanrauh** [comment]
Good idea. Thank you very much for caring about this!

**Comment by stephanrauh** [comment]
Your bugfixes have landed with version 23.0.0-alpha.2.

Enjoy!
Stephan

**Comment by Agesse** [comment]
Thank you for being so reactive. We use your reader in some french agencies and we have strict accessibility legislation here, I'm really happy to be able to contribute directly on your projet.
